ledger/participant-integration-api/src/main/scala/platform/apiserver/RateLimitingInterceptor.scala

```diff
@@ -12,7 +12,13 @@ import com.daml.error.definitions.LedgerApiErrors.{
 }
 import com.daml.error.{ContextualizedErrorLogger, DamlContextualizedErrorLogger}
 import com.daml.metrics.{MetricName, Metrics}
-import com.daml.platform.apiserver.RateLimitingInterceptor._
+import com.daml.platform.apiserver.RateLimitingInterceptor.{
```

Comment:
Why do we still need this version of the RateLimitingInterceptor?

Reply:
This maybe a rebase recovery - removed.